src/App.css

```diff
@@ -8,4 +8,11 @@
 
 .card {
   padding: 2em;
+}
+
+.grid {
```

Comment:
I would add another property to align the items in  the middle of the page: 

justify-content: center;

Reply:
Ok. done

Reply:
The reason the modal and carousel doesnt work anymore is that wrapping the whole product card in link makes everything whne clicked go to the the place id and overides the click logic inside the the product card. this can be changed in the future.

your changes look good, now th epage has some shape! good work